### ISSUE TITLE
set cache-control header for sitemap.xml

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#16929]
when we serve the (non-`?debug`) `/sitemap.xml` we should set the `cache-control` header so frequent retries are handled by caching.

The value must take into account we'll be refreshing the content served by `/sitemap.xml` on a cron job, likely daily - i.e. if we set a value of 24 hours sitemap could potentially be out of date by almost 48 hours.

**Comment by AlexandraMoga** [comment]
@eviljeff can this be verified individually in any way or should it be covered when testing for #16508 becomes available?

**Comment by AlexandraMoga** [comment]
> when we serve the (non-`?debug`) `/sitemap.xml`

@eviljeff does this mean that this issue can't be verified against the debug version of the sitemap?

**Comment by eviljeff** [comment]
@AlexandraMoga yes.  `?debug` isn't supposed to be cached so doesn't support those headers
